packages/stark-ui/src/modules/table/components/_table.component.scss

```diff
@@ -19,17 +19,20 @@
       display: flex;
       justify-content: flex-end;
 
-      &.actions-alt {
-        flex-direction: row-reverse;
-      }
-
       stark-minimap {
         mat-icon {
           width: 18px;
           height: 18px;
         }
       }
     }
+
+    .stark-table-rows-counter {
+      flex: 1;
+      text-align: center;
+      line-height: 35px;
```

Comment:
In my opinion this `line-height` could be lower, maybe 22px?
![rows counter small screens 2](https://user-images.githubusercontent.com/5698318/58081700-ca02f800-7bb5-11e9-86e0-fb27bfaf5435.PNG)

Cause right now with 35px it looks a bit weird:
![rows counter small screens](https://user-images.githubusercontent.com/5698318/58081737-dbe49b00-7bb5-11e9-83dc-09ae372896b6.PNG)

Reply:
Yep', it's OK for me. I took the value we had before.